deploy/crds/performance.openshift.io_performanceprofiles_crd.yaml

```diff
@@ -54,6 +54,11 @@ spec:
                   description: Isolated defines a set of CPUs that will be used to
                     give to application threads the most execution time possible,
                     which means removing as many extraneous tasks off a CPU as possible.
+                    It important to notice, that the CPU manager can choose any CPU
+                    to run the workload, except reserved CPUs, so to guarantee that
+                    your workload will run on the isolated CPU, reserved and isolated
+                    CPUs should include all online CPUs and isolated CPUs field should
```

Comment:
online ? or all CPUs ?

Reply:
Good point. Thinking about SMT support, I'd say "online". AFAIK there is no other case on which `online_cpus != installed_cpus`. Also, we don't expect cpus to be put online/offline during the worker lifecycle, right?
cc @MarSik to doublecheck.